### PR TITLE
Add option for custom domain to single sign on

### PR DESCRIPTION
**Pull request by chaselee**
- Adds `domain` argument to `Mbsy.SingleSignOn.embed_html`
- Falls back to existing `username.getambassador.com` behavior

Requested in https://github.com/GetAmbassador/mbsy/issues/19.

### CODE REVIEW

**Comment by chaselee** [comment]
@brandoshmando this would also need a version bump

**Comment by brandoshmando** [comment]
reviewing

**Comment by brandoshmando** [comment]
[ROMWOD, LLC - Custom Domain/SSO with Ruby gem](https://trello.com/c/KQGKdAqs/582-romwod-llc-custom-domain-sso-with-ruby-gem)

File: lib/mbsy/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Mbsy
-  VERSION = '2.0.0'
+  VERSION = '2.0.1'
```
Comment (chaselee) [inline lib/mbsy/version.rb]: @brandoshmando this should be 2.1.0 if we're following semver since it's a backwards-compatible enhancement.

**Pull request closed without merging**